server/channels/app/scheduled_post.go

```diff
@@ -38,7 +39,13 @@ func (a *App) SaveScheduledPost(rctx request.CTX, scheduledPost *model.Scheduled
 		return nil, model.NewAppError("App.ScheduledPost", "app.save_scheduled_post.save.app_error", map[string]any{"user_id": scheduledPost.UserId, "channel_id": scheduledPost.ChannelId}, "", http.StatusBadRequest)
 	}
 
-	// TODO: add WebSocket event broadcast here
+	message := model.NewWebSocketEvent(model.WebsocketScheduledPostCreated, "", scheduledPost.ChannelId, "", nil, connectionId)
```

Comment:
We do want to specify the user ID here as we only want to send this websocket event to the user who created the scheduled posts, to all their clients except the one with the connection ID.